doc/user/separate.xml

```diff
@@ -17,98 +23,111 @@
+  separate from the source files. Two main benefits are
+  the ability to have different configurations simultaneously
+  without build conflicts, and being version-control friendly.
 
-  The above copyright notice and this permission notice shall be included
-  in all copies or substantial portions of the Software.
+  </para>
 
-  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY
-  KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
-  WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
-  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
-  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
-  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
-  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+  <para> 
+
+  Consider if you have a project to build an embedded
+  software system for a variety of different controller hardware.
+  The system is able to share a lot of code,
+  so it makes sense to use a common source tree,
+  but certain build options in the source code
+  and header files differ. For a regular in-place build,
+  the build outputs go in the same place as the source code.
+  If you build <emphasis>Controller A</emphasis> first,
+  followed by <emphasis>Controller B</emphasis>,
+  on the <emphasis>Controller B</emphasis> build everything that
+  uses different build options has to be rebuilt since those
+  objects will be different
+  (the build lines, including preprocessor defines, are part of
+  &SCons;'s out-of-date calculation for this reason).
+  If you go back and build for <emphasis>Controller A</emphasis> again,
+  things have to be rebuilt again for the same reason.
+  However, if you can separate the locations of the output files,
+  so each controller has its own location for build outputs,
+  this problem can be avoided.
 
--->
+  </para>
 
   <para>
 
-  It's often useful to keep any built files completely
-  separate from the source files. Consider if you have a
-  project to build software for a variety of different
-  controller hardware.  The boards are able to share a
-  lot of code, so it makes sense to keep them in the same
-  source tree, but certain build options in the source code
-  and header files differ.  If you build "Controller A" first,
-  then "Controller B", on the "Controller B" build everything would
-  have to be rebuilt, because &SCons; recognizes that the build
-  instructions are different from those used in the "Controller A"
-  build for each target - the build instructions are part of
-  &SCons;'s out-of-date calculation.
-  Now when you go back and build for "Controller A",
-  things have to be rebuilt from scratch again for the same reason.
-  However, if you can separate the locations of the output files,
-  this problem can be avoided.
-  You can even set up to do both builds in one invocation of &SCons;.
+  Having a separated build also helps you keep your source tree clean -
```

Comment:
..build directory.. ? instead of just build?

Reply:
build tree ?

Reply:
I don't mind "build tree", that seems to match the sentence.  Shall I wait for other comments before adjusting?

Reply:
build tree it is then!

Reply:
Updated.